ocaml/test/test_common.ml

```diff
@@ -60,7 +60,7 @@ let make_localhost ~__context =
 	   simple thing first and just set localhost_ref instead. *)
 	(* Dbsync_slave.refresh_localhost_info ~__context host_info; *)
 	Xapi_globs.localhost_ref := Helpers.get_localhost ~__context;
-	Create_misc.ensure_domain_zero_records ~__context host_info;
+	Create_misc.ensure_domain_zero_records ~__context ~host:Xapi_globs.localhost_ref host_info;
```

Comment:
This is missing a `!`. Should be `!Xapi_globs.localhost_ref` since this is a ref type, assigned above using `:=`, not `=`. This is the reason for the build failure:

```
File "test_common.ml", line 63, characters 57-81:
Error: This expression has type [ `host ] Ref.t ref
       but an expression was expected of type [ `host ] Ref.t
```